scripts/vsmlrt.py

```diff
@@ -467,16 +467,29 @@ class RealESRGANModel(enum.IntEnum):
     animevideo_xsx4 = 1
     # v3
     animevideov3 = 2 # 4x
+    # janai (2x)
```

Comment:
should be labeled as contributed models, and add the link to the original repository and maintainer